README.md

```diff
@@ -72,13 +73,22 @@ from pyartifactory import Artifactory
 art = Artifactory(url="ARTIFACTORY_URL", auth=('USERNAME','PASSORD_OR_API_KEY'), cert="/path_to_file/server.pem",api_version=1)
 ```
 
+Specify a local cert to use as custom CA certificate
+
+```python
+from pyartifactory import Artifactory
+art = Artifactory(url="ARTIFACTORY_URL", auth=('USERNAME','PASSORD_OR_API_KEY'), verify="/path_to_file/ca.pem", api_version=1)
```

Comment:
Why did you remove the 

The `verify` controls whether SSL/TLS certificate validation is performed for the server's certificate, while `cert` is used to provide a client-side certificate for mutual authentication when required by the server. These parameters are both related to SSL/TLS security but serve different purposes in making secure HTTP requests with the requests library.

They both must be documented. It is actually possible to set a [cert](https://github.com/anancarv/python-artifactory/blob/master/pyartifactory/models/auth.py#L16) key

Reply:
Done. 

@anancarv, @sigma67 please let me know if everything is now ok for you. Cheers!

Reply:
LGTM, you'll only need to resolve the conflicts and it's gonna be good to be merged

Reply:
Already done!